nicolegooden/ideabox-boilerplate
# Jv iteration 3 refactor

What’s this PR do?
- Clean up extra code and placeholders in main.js
- Fix breakpoints and media queries in styles.css
- index.HTML: Remove template cards 

  Where should the reviewer start?  
- In the browser

How should this be manually tested?  
- Check break points
- Reload page to make sure template cards do not appear

Any background context you want to provide?  
N/A
What are the relevant tickets?  
N/A

Screenshots (if appropriate)

  Questions: 
How can we get the comment margin to disappear?
